helpers/securewebhook/securewebhook_test.go

```diff
@@ -40,3 +49,55 @@ func TestSecureWebhookGetRequestBody(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, expectedJSONEnabled, actualJSON, fmt.Sprintf("SecureWebhook.Enable should be '%b', got %b", expectedJSONEnabled, actualJSON))
 }
+
+func TestVerifySignature(t *testing.T) {
+	priKey, err := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
+	require.NoError(t, err)
+
+	timestamp := strconv.FormatInt(time.Now().Unix(), 10)
+
+	payload, err := json.Marshal(map[string]interface{}{
+		"event":      "test_event",
+		"category":   "example_payload",
+		"message_id": "message_id",
+	})
+	require.NoError(t, err)
+
+	h := sha256.New()
+	h.Write([]byte(timestamp))
+	h.Write(payload)
+
+	r, s, err := ecdsa.Sign(rand.Reader, priKey, h.Sum(nil))
```

Comment:
This test is conflating the `sign` and `verify` implementations a bit (i.e., if there is a bug in some common code between those implementations then this test could still pass). I'd rather have concrete sample data: payload, public key, signature, and possibly timestamp. This sample data will be used for every helper library to prove that the verification code is correct without needing a dependency on signing in each (sign once elsewhere, verify everywhere).